src/screens/courses/CardScreen/index.tsx

```diff
@@ -25,6 +25,8 @@ const CardScreen = ({ index, exitConfirmationModal, setExitConfirmationModal, go
   const [isLeftSwipeEnabled, setIsLeftSwipeEnabled] = useState<boolean>(true);
   const [isRightSwipeEnabled, setIsRightSwipeEnabled] = useState<boolean>(false);
 
+  if (isWeb) enableLegacyWebImplementation(true);
```

Comment:
Peux tu mettre un peu de contexte autour de cette option de react-native-gesture-handler et pourquoi ca resoud notre pb ?

Reply:
depuis Gesture Handler 2.10.0 une nouvelle implémentation Web est activée par défaut. Or react-native-drax utilise une vieille version de Gesture Handler (antérieure à la version 2.0.0) donc ne supporte sans doute pas cette nouvelle implémentation. Cette commande permet d'utiliser l'ancienne implémentation compatible avec react-native-drax. 

Edit : je me dis que c'est pas forcément une bonne idée d'utiliser quelque chose qui est en phase d'être déprécié. Par ailleurs, si ça fonctionne très bien sur desktop j'ai l'impression que sur web mobile ça pose quelque soucis, je vais peut-être le retirer

Reply:
Je suis d'accord avec toi, en effet en lancant le web sur un mobile le drag and drop ne foncitonne pas